Exercises/functions/functions.js

```diff
@@ -1,3 +1,40 @@
+/*eslint func-names: ["error", "never"]*/
+/*eslint-disable no-console*/
+//1. Create a function declaration, function expression, lambda function, and using function object.
+//function declaration
+function getName(obj) {
+  return obj.name || 'Anonymus';
+}
+console.log(getName({ name: 'Olha' }));
+
+//function expression
+const getMainName = function (obj) {
+  return (obj && obj.name) || 'Anonymus';
+};
+console.log(getMainName({ name: 'Michael' }));
+
+//lamda function and using function object
+const getObjName = obj => (obj && obj.name) || 'Anonymus';
+console.log(getObjName({ name: '' }));
+
+//2. Get parameters amount of function `takeMyParametersAmount`.
 function takeMyParametersAmount(a, b, c) {
```

Comment:
meant to take the number of parameters which taking function, I thin I missed it on the lesson, so the tip is, that function has length property and it represent number of parameters it takes

Reply:
thanks! Got it! Updated the code using function.lenght 
return takeMyParametersAmount.length;